packages/express/src/accountsExpress.js

```diff
@@ -22,6 +22,22 @@ const accountsExpress = (AccountsServer) => {
   ? res.status(500).jsonp(err.serialize().message)
   : res.status(500).jsonp({ message: err.message }));
 
+  router.use(async (req, res, next) => {
+    const accessToken = get(req.headers, 'accounts-access-token', undefined) || get(req.body, 'accessToken', undefined);
+    if (!isEmpty(accessToken)) {
+      try {
+        const user = await AccountsServer.resumeSession(accessToken);
+        // eslint-disable-next-line no-param-reassign
+        req.user = user;
+        // eslint-disable-next-line no-param-reassign
+        req.userId = user.id;
+      } catch (e) {
+        console.log('Failed to resume session');
```

Comment:
I don't think I want to do that here because failing to resume a session shouldn't stop the request, it just refrains from assigning req.user